src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    static String getRandomBall() {
```

Comment:
Avoid using static methods for functionality like this. It's better to create an instance of the Lottery class and then call the method on that instance.